kevm

```diff
@@ -252,16 +251,6 @@ run_interpret() {
     esac
 }
 
-run_solc() {
-    local contract_name json_run_file
-
-    contract_name="$1" ; shift
-    json_run_file="${run_file}.json"
-
-    execute solc --combined-json abi,bin-runtime,storage-layout,hashes "${run_file}" > "${json_run_file}"
-    execute python3 -m kevm_pyk solc-to-k "$@" ${backend_dir}/*-kompiled "${run_file}" "${json_run_file}" "${contract_name}"
-}
-
```

Comment:
We should still have this command present in `kevm` runner script, in anything just as a wrapper to `kevm_pyk ...` which invokes it with the correct arguments (like passing it the `*_kompiled` directory appropriately).